code/modules/mob/living/silicon/silicon_defense.dm

```diff
@@ -61,11 +61,38 @@
 			M.visible_message("<span class='notice'>[M] pets [src]!</span>", \
 							"<span class='notice'>You pet [src]!</span>")
 			playsound(loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
-		if("grab")
+		if(INTENT_GRAB)
 			grabbedby(M)
+		if(INTENT_DISARM)
+			tip_over(M)
 		else
 			M.do_attack_animation(src, ATTACK_EFFECT_PUNCH)
 			playsound(loc, 'sound/effects/bang.ogg', 10, 1)
 			visible_message("<span class='notice'>[M] punches [src], but doesn't leave a dent.</span>", \
 						"<span class='notice'>[M] punches [src], but doesn't leave a dent.</span>")
 	return FALSE
+
+/*
+Rotates a silicon's sprite 180 degrees, returns true if successful, false otherwise. All of the un-tipping is handled by the tilted component
```

Comment:
Already using multiline
Its getting a tiny bit long, can you make each sentence a new line?